SPECS/kernel/kernel-uki-dracut.conf

```diff
@@ -0,0 +1,38 @@
+# generic + compressed please
+hostonly="no"
+compress="xz"
+
+# VMs can't update microcode anyway
+early_microcode="no"
+
+# modules: basics
+dracutmodules+=" base systemd systemd-initrd dracut-systemd dbus usrmount shutdown "
+
+# modules: storage support
+dracutmodules+=" dm rootfs-block fs-lib "
+
+# modules: tpm and crypto
+dracutmodules+=" crypt crypt-loop tpm2-tss "
+
+# modules: support root on virtiofs
+dracutmodules+=" virtiofs "
+
+# modules: use sysext images (see 'man systemd-sysext')
+dracutmodules+=" systemd-sysext "
+
+# drivers: virtual buses, pci
+drivers+=" virtio-pci virtio-mmio "      # qemu-kvm
```

Comment:
These drivers are in addition to the base set required by defaults.conf packaged in dracut, yes?

Reply:
This original config is here: https://src.fedoraproject.org/rpms/kernel/blob/rawhide/f/dracut-virt.conf.

I don't think we are using `defaults.conf` for our UKI.

Reply:
This is inconsequential because it is only used during rpmbuild time and neither packaged nor installed on a target system.

Reply:
In other words, this configuration is isolated to the initrd built, added to the UKI image which is okay.